examples/realtime.rs

```diff
@@ -45,10 +39,41 @@ fn main() {
         .unwrap();
     stream.play().unwrap();
 
-    let mut signal = scene_handle.control::<oddio::SpatialScene, _>().play(
-        oddio::FramesSignal::from(boop),
-        [-speed, 10.0, 0.0].into(),
-        [speed, 0.0, 0.0].into(),
+    // Let's make some audio.
+    // Here, we're just manually constructing a stream of sounds.
+    // in `oddio`, a sound like this is called `Frames` (each sample being a `Frame`).
```

Comment:
```suggestion
    // Here, we're manually constructing a sound, which might otherwise be e.g. decoded from an mp3.
    // in `oddio`, a sound like this is called `Frames` (each frame consisting of one sample per channel).
```

A sample is a single amplitude measurement, whereas a frame has as many samples as there are channels in use.

Reply:
ahhh thanks. channels trip me up/surprise me often